data/events.txt

```diff
@@ -1382,6 +1382,14 @@ event "initial deployment 4"
 		fleet "Large Southern Pirates" 2000
 		fleet "Small Free Worlds" 3000
 		fleet "Large Free Worlds" 5000
+	system "Shaula"
+		fleet "Small Southern Pirates" 500
+		fleet "Large Southern Pirates" 800
+		fleet "Small Core Pirates" 4000
+		fleet "Large Core Pirates" 7000
+		fleet "Small Northern Pirates" 5000
+		fleet "Large Northern Pirates" 9000
+		fleet "Large Militia" 9000
```

Comment:
I think that was meant to be Free Worlds, but I haven't looked at this in a long time, and I can't remember.